lib/helpers/email.js

```diff
@@ -5,7 +5,7 @@ import fs from 'fs';
 import htmlToText from 'html-to-text';
 
 const defaultMailOptions = {
-  from: 'Learning Locker <noreply@learninglocker.net>',
+  from: `${process.env.LL_FROM_NAME} <${process.env.LL_FROM_EMAIL}>` || 'Learning Locker <noreply@learninglocker.net>'
```

Comment:
Not sure this is going to work as expected, as empty values on both would lead to an string of `" "` which would not be falsey. Perhaps look at `lodash/defaultTo` instead?

Reply:
The problem doing it similarly to some of the other variables is you don't want to introduce another required default in the config file.   

Is there an example of how you've used defaultTo elsewhere in the code?